podcasts/Theme.swift

```diff
@@ -181,15 +201,28 @@ class Theme: ObservableObject {
     }
 
     class func preferredLightTheme() -> ThemeType {
+
+        if FeatureFlag.settingsSync.enabled {
+            return SettingsStore.appSettings.lightThemePreference
+        }
+
         let savedType = UserDefaults.standard.integer(forKey: preferredLightThemeKey)
 
-        guard let themeType = ThemeType(rawValue: savedType) else { return .dark }
+        guard let oldTheme = ThemeType.Old(rawValue: savedType) else { return .dark }
+
+        let themeType = ThemeType(old: oldTheme)
 
         return themeType
     }
 
     class func setPreferredLightTheme(_ preferredType: ThemeType, systemIsDark: Bool) {
-        UserDefaults.standard.setValue(preferredType.rawValue, forKey: preferredLightThemeKey)
+
+        if FeatureFlag.settingsSync.enabled {
+            SettingsStore.appSettings.lightThemePreference = preferredType
+        } else {
+            UserDefaults.standard.setValue(preferredType.rawValue, forKey: preferredLightThemeKey)
+        }
+        UserDefaults.standard.setValue(preferredType.old.rawValue, forKey: preferredLightThemeKey)
```

Comment:
Same as the other comment. The `else` will always be overriden by the last `setValue`.